assets/tests/components/notificationBellIcon.test.tsx

```diff
@@ -115,4 +138,31 @@ describe("NotificationBellIcon", () => {
       .toJSON()
     expect(tree).toMatchSnapshot()
   })
+
+  test("renders when there are new detour notifications and user is part of DetoursList group", () => {
+    jest.mocked(getTestGroups).mockReturnValue([TestGroups.DetoursList])
+
+    const state = { ...initialState, openView: OpenView.None }
+    const { baseElement } = render(
+      <StateDispatchProvider state={state} dispatch={jest.fn()}>
+        <NotificationsContext.Provider value={unreadDetourNotificationState}>
+          <NotificationBellIcon />
+        </NotificationsContext.Provider>
+      </StateDispatchProvider>
+    )
+
+    expect(baseElement).toMatchSnapshot()
+  })
+
+  test("renders when there are new detour notifications and user is not part of DetoursList", () => {
+    const state = { ...initialState, openView: OpenView.None }
```

Comment:
https://github.com/mbta/skate/pull/2819/files#r1783392531
```suggestion
    const state = stateFactory.build({ openView: OpenView.None })
```

Reply:
https://github.com/mbta/skate/pull/2819/commits/35dcb01a1d2f998e202202651a7f46494e5671e0